scripts/lspArtifact.ts

```diff
@@ -158,7 +158,10 @@ export async function downloadLanguageServer(): Promise<void> {
                         if (!fs.existsSync(path.join(resourcesDir, 'servers', 'aws-lsp-codewhisperer.js'))) {
                             throw new Error(`Extracting aws-lsp-codewhisperer.js failure`)
                         }
-                        if (!fs.existsSync(path.join(resourcesDir, 'clients', 'amazonq-ui.js'))) {
```

Comment:
You need to make it compatible with https://github.com/aws/aws-toolkit-vscode/pull/7421/files.  Please check function `export function getBundledResourcePaths(ctx: vscode.ExtensionContext): AmazonQResourcePaths {`